evelyn_chavez/module_practive/bank_account/bank_account.rb

```diff
@@ -0,0 +1,68 @@
+class BankAccount
+    attr_accessor :account_number, :checking_balance, :savings_balance
+    @@no_of_accounts = 0
+
+    def initialize
+        @checking_balance = 2500
+        @savings_balance = 5000
+        @@no_of_accounts += 1
+        puts "Number of accounts #{@@no_of_accounts}"
+    end
+
+    def account_balance
+        puts "Your total account balance is #{@checking_balance + @savings_balance}"
+        puts "Your total checking balance is #{@checking_balance}"
+        puts "Your total savings balance is #{@savings_balance}"
+    end
+
+    def deposit(num, account)
+        if account == "checking"
+            puts "deposit #{num} into checking"
+            @checking_balance = num + @checking_balance
+            puts "your new checking balance is  #{@checking_balance}"
+        elsif
+            account == "savings"
+            puts "deposit #{num} into savings"
+            @savings_balance = num + @savings_balance
+            puts "your new saving balance is #{@savings_balance}"
+        end
+    end
+    
+    def withdraw(num, account)
+        if account == "savings"
+            if num < @savings_balance
+                @savings_balance -= num
+                
+			else
+                puts "insufficient fund, cannot withdraw from savings"
+			end
+        elsif
```

Comment:
specifically look for checking. We don't want to land here if its not a checking account(eg the input can be "general", "shared") and it will still reduce money from checking account.